src/base/orgs/View/Projects.svelte

```diff
@@ -40,36 +43,65 @@
 </style>
 
 <div class="projects">
-  {#await getProjects()}
-    <Loading center />
-  {:then projects}
-    {#each projects as project}
-      <div class="project">
-        {#if "safeTxHash" in project} <!-- Pending project -->
-          <Widget {project} org={org.address} {config} faded>
-            <span slot="stateHash">
-              <span class="mobile">commit {formatCommit(project.anchor.stateHash)}</span>
-              <span class="desktop">commit {project.anchor.stateHash}</span>
-            </span>
-            <span class="anchor" slot="actions">
-              {#if org.safe && account}
-                <Anchor {project} safe={org.safe} on:success={() => updateRecords()} {account} {config} />
-              {/if}
-            </span>
-          </Widget>
-        {:else} <!-- Anchored project -->
-          <Widget {project} org={org.address} {config}>
-            <span slot="stateHash">
-              <span class="mobile">commit {formatCommit(project.anchor.stateHash)}</span>
-              <span class="desktop">commit {project.anchor.stateHash}</span>
-            </span>
-          </Widget>
-        {/if}
-      </div>
-    {/each}
-  {:catch err}
-    <Message error>
-      <strong>Error: </strong> failed to load projects: {err.message}.
-    </Message>
-  {/await}
+  {#if profile.org}
```

Comment:
Okay this is to fetch all the org projects and else query the `profile.anchorsAccount` projects..
We could also separate them into separate components.. eg. `PendingProjects` `AnchoredProjects`